playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.jsx

```diff
@@ -58,6 +60,8 @@ const Typeahead = (props: Props) => {
   }
 
   if (typeof(props.loadOptions) === 'string') selectProps.loadOptions = get(window, props.loadOptions)
+  if (typeof(props.getOptionLabel) === 'string') selectProps.getOptionLabel = get(window, props.getOptionLabel)
+  if (typeof(props.getOptionValue) === 'string') selectProps.getOptionValue = get(window, props.getOptionValue)
```

Comment:
Just like `loadOptions`, we need to check for the presence of the prop. If this pattern emerges in another kit, we should abstract this away to a utility method for re-use.